index.js

```diff
@@ -1,2 +1,6 @@
+const input = document.getElementById("button");
+
 function addingEventListener() {
+  alert("I was clicked!");
+  input.addEventListener("click", addingEventListener);
```

Comment:
Be careful! In this case, you actually created a loop. When the event listener is triggered, it will add a new event listener each time, meaning it will stack each time the input is clicked. Compare what you have to something like this. 

```js
function addingEventListener() {
  const input = document.getElementById("button");

  function clickAlert() {
    alert("I was clicked!");
  }

  input.addEventListener("click", clickAlert);
}
  ```